rollup/plugins/bundlePlugins.js

```diff
@@ -116,9 +116,10 @@ export function makeTerserPlugin() {
           '_driver',
           '_initStorage',
           '_support',
-          // We want to keept he _replay and _isEnabled variable unmangled to enable integration tests to access it
+          // We want to keept some replay fields unmangled to enable integration tests to access them
```

Comment:
Just fixing my own typo :D

```suggestion
          // We want to keep some replay fields unmangled to enable integration tests to access them
```